.changeset/fluffy-pillows-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@neo4j/graphql": patch
+---
+
+Setting resolvable to false no longer prevents queries and mutations for a type from being generated.
```

Comment:
Probably worth being a bit more specific so users can more easily see if this will affect them

```suggestion
Federation: Setting `@key` resolvable to false no longer prevents queries and mutations for a type from being generated.
```